test/models/term_test.rb

```diff
@@ -299,6 +299,32 @@ class TermTest < ActiveSupport::TestCase
     end
   end
 
+  test 'categorized scope returns an active record relation' do
+    assert_kind_of ActiveRecord::Relation, Term.categorized
+  end
+
+  test 'categorized scope accounts for terms with multiple categorizations' do
+    categorized_count = Term.categorized.count
+    t = terms('doi')
+
+    term_category_count = t.categorizations.count
+
+    # term has been categorized already
+    assert_operator 1, :<=, term_category_count
+
+    new_record = {
+      term: t,
+      category: categories('navigational'),
+      confidence: 0.5,
+      detector_version: '1'
+    }
+    Categorization.create!(new_record)
+
+    # The term has gained a category, but the categorized scope has not changed size.
+    assert_operator term_category_count, :<, t.categorizations.count
```

Comment:
No change necessary, but `term_category_count` as a variable name made me have to think more than felt necessary as to what is going on here. Maybe something like `orig_category_count` as it gets compared to the same count after a categorization is made might be better?

Reply:
Yeah, I can see how a different name might be better. I've tried to make it something clearer, and I particularly like the `orig_` prefix.